src/utils/currency/tokenPriceByName.ts

```diff
@@ -80,14 +81,14 @@ export const fetchTokenPriceByName = async ({
         );
       }
 
-      console.error(
-        `Unable to get price for ${tokenName}. It probably doesn't have a listed exchange value.`,
+      debugLogging(
+        `CoinGecko: Unable to get price for ${tokenName}. It probably doesn't have a listed exchange value.`,
       );
       return 0;
     });
   } catch (e) {
     if (import.meta.env.DEV) {
```

Comment:
I think it no longer makes sense to have this condition given we use `debugLogging`

Reply:
Very good point. I'll remove them!